requirements/dev_python27.txt

```diff
@@ -1,11 +1,2 @@
--r base-py2.txt
-
-mock>=2.0.0
-apache-libcloud>=0.14.0
-boto>=2.32.1
-boto3>=1.2.1
-moto>=0.3.6
-SaltPyLint>=v2017.3.6
-pytest>=3.5.0
-git+https://github.com/eisensheng/pytest-catchlog.git@develop#egg=Pytest-catchlog
```

Comment:
I don't see the pytest-catchlog in the new dev.txt file. I am not sure what that particular addition to the Py27 requirements file is for since there's no comment about it, but I wanted to double check that this was intentional to leave that out.

Reply:
pytest-catchlog has been merged into pytest upstream, so we no longer needed it specified.

Reply:
Perfect!